Added opacity slider description (#385)
opengisch/QField-docs: documentation/how-to/map-interaction.en.md

- *Reload data* to get the current data of a layer with remote sources.
- *Show feature list* to show all the layer's features in the identification list.

> Additionally, you have the option to adjust the layer opacity for each individual layer using the "Opacity" slider.

Reviewer: Just a suggestion for this phrase 
```suggestion
Also, you can change how transparent each layer is by moving the "Opacity" slider.
```
